modules/python/src2/cv2.cpp

```diff
@@ -27,7 +27,7 @@
 #pragma warning(pop)
 #endif
 
-#define MODULESTR "cv2"
+#define MODULESTR "_cv2"
```

Comment:
This renaming is not harmless in general.
It makes sense to keep existed flow for current release series.

I believe it is not a problem to load whole package at once. No need to support on-demand loading of submodules for conflict "subnames" already existed in C binary extension.

BTW, we have discussions about "opencv" name for package in 5.0 scope.